build.py

```diff
@@ -124,9 +124,30 @@ def run(config, start_class_path, arguments):
 ### def run ###
 
 
+# USAGE
+#
+# Print basic usage info.
+#
+def usage(args) :
+  print('Unknown command "%s"' % args)
```

Comment:
This would also print "Unknown command: help". And it will have trouble if len(args) == 0. Let's move this line to the elif at line 150